.github/workflows/ci.yml

```
@@ -225,8 +225,8 @@ jobs:
           TYPESCRIPT_ESLINT_EXPERIMENTAL_TSSERVER: true
 
   website_tests:
-    # The NETLIFY_TOKEN secret will not be available on forks
-    if: github.repository_owner == 'typescript-eslint'
+    # The NETLIFY_TOKEN secret will not be available on PRs from forks
+    if: (github.event_name == 'push' && github.ref_name == 'main') || (github.event.pull_request.head.repo.full_name == github.repository)
```

I have a question, `merge_group` should trigger this test? Then it may be following

```suggestion
    if: contains(fromJSON('["push", "merge_group"]'), github.event_name) || (github.event.pull_request.head.repo.full_name == github.repository)
```
```

---

TIL: https://github.blog/changelog/2022-08-18-merge-group-webhook-event-and-github-actions-workflow-trigger

We don't use merge queues. I don't think test triggers need to account for them.

---

Thanks for sharing the direction! I just had confused with the trigger in here 🙇‍♂️  https://github.com/typescript-eslint/typescript-eslint/blob/a56af12b83fab9def8989a83d5b9d2a660b099fd/.github/workflows/ci.yml#L10